common/lib/xmodule/xmodule/js/src/sequence/display.js

```diff
@@ -448,6 +451,23 @@
             this.el.find('.nav-item.active .bookmark-icon-sr').text('');
         };
 
+        /**
+         * Sends the vertical+block id to Google Analytics,
+         * in order to track by unit item.
+         */
+        Sequence.prototype.sendPageViewToGA = function(position) {
+            var element = this.link_for(position);
```

Comment:
What happens if the position is not valid ?

Reply:
Must it fails, and it's not necessary to send GA event. See 415 line. @andrey-canon